### PR TITLE
Incorpore navigation

### PR DESCRIPTION
**Pull request by andymotagl**
## Description

Integrate the current components in a proper navigation. Added a button to achieve this. Fixes the identified issues when testing all the components.

### CODE REVIEW

File: web/src/components/Init.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+  const { loginWithRedirect } = useAuth0();
 
-    <p className="lead">
-      This is a sample application with an authentication flow
-    </p>
-  </div>
-);
+  return (
+    <div className="text-center Init my-5">
+      <h1 className="mb-4">Galielei Learning App</h1>
```
Comment (henrymcarvajal-galilei) [inline web/src/components/Init.js:10]: Typo in GaliElei

File: web/src/utils/dates.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const formatDate = (input) => {
```
Comment (henrymcarvajal-galilei) [inline web/src/utils/dates.js:1]: Experience PR has a formatDate that accounts for timezone, so I think we can resort to that function when merge is complete

File: web/src/views/InviteLeader.js [hunk 3]
```diff
@@ -151,13 +153,17 @@ const InviteLeader = () => {
...
+                <ReturnButton to={'/landing'}/>
+            </ButtonGroup>
 
             {invitationDetails && invitationDetails.hasInvitationCapacity && (
                 <Alert>
                     <h4 className="alert-heading">
                         {invitationDetails.name} tiene un token de invitación generado!
                     </h4>
-                    <p>Ten en cuenta que este token de invitación venecera el {new Date(invitationDetails.expiration_token).toISOString().replace("T"," ").substring(0, 19)}, y tiene {invitationDetails.remainingInvitations} plazas de invitación restantes.</p>
+                    <p>Ten en cuenta que este token de invitación venecera el {formatDate(invitationDetails.expiration_token)}, y tiene {invitationDetails.remainingInvitations} plazas de invitación restantes.</p>
```
Comment (henrymcarvajal-galilei) [inline web/src/views/InviteLeader.js:165]: typo & missing tilde in venEcerÁ

File: web/src/views/InvitePlayer.js [hunk 4]
```diff
@@ -158,7 +161,7 @@ const InvitePlayer = () => {
                     <h4 className="alert-heading">
                         {invitationDetails.name} tiene un token de invitación generado!
                     </h4>
-                    <p>Ten en cuenta que este token de invitación venecera el {new Date(invitationDetails.expiration_token).toISOString().replace("T"," ").substring(0, 19)}, y tiene {invitationDetails.remainingInvitations} plazas de invitación restantes.</p>
+                    <p>Ten en cuenta que este token de invitación venecera el {formatDate(invitationDetails.expiration_token)}, y tiene {invitationDetails.remainingInvitations} plazas de invitación restantes.</p>
```
Comment (henrymcarvajal-galilei) [inline web/src/views/InvitePlayer.js:164]: Same typos as above

**Review by henrymcarvajal-galilei — APPROVED** [review]
At last, integrated navigation!!!... 🤭

**Review by sebastian-motavita — APPROVED** [review]
Just minor adjustments

**Pull request merged**